### DISCUSSION TITLE
AppxRecipe and Transitive nuget packages

### DISCUSSION THREAD

**Discussion by Daniellled** [microsoft/WindowsAppSDK#1412] [Q&A]
We have a nuget package that contains both debug and release assemblies. In the debug assemblies we do something special that the release assemblies excludes. The nuget package uses the buildTransitive nuget feature to execute a target that makes sure upon debug builds we move the correct dll to the output directory. 

https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files
https://github.com/NuGet/Home/wiki/Allow-package--authors-to-define-build-assets-transitive-behavior

When we compile a MAUI WinUI project the debug dlls end up in the correct directories. Debug is in the debug directory, and release is in the release directory.

However, when debugging a MAUI WinUI application we are noticing that the appxmanifest is always pointing to the release binaries. This is causing the release binary from our nuget package to show up in the AppX folder.

What target should our nuget package override to correct this problem?

**Comment by btueffers** [comment]
Hi @Daniellled, is this the same issue you're facing? https://github.com/microsoft/WindowsAppSDK/issues/1469
